Add testnet support

ananas-block opened the pull request:
Changes:
- updated program ids to testnet
- updated accounts pubkeys and json files for tests which depend on program ids
- replaces .rpc calls in MerkleTreeConfig calls with sendAndConfirmTransaction - had to adjust the merkle tree tests for different error propagation
- added ts script for 100 sol airdrops from testnet
- added ts script which instantiates a look up table - can be deleted once cli is ready

SwenSchaeferjohann commented on light-zk.js/src/relayer.ts:
```diff
@@ -107,7 +107,7 @@ export class Relayer {
   ): Promise<RelayerSendTransactionsResponse> {
     try {
-      const response = await axios.post(this.url + "/relayInstruction", {
+      const response = await axios.post(this.url + "/relayTransaction", {
```
arent we technically still sending instruction**s** ?

ananas-block replied:
Yes we are sending several instructions which in concert execute one shielded transaction.
Every instruction is executed in a single solana transaction.

happy for suggestions on better naming!

SwenSchaeferjohann replied:
probably `relayInstructions` makes most sense then? but this is not urgent so can still merge this pr now

SwenSchaeferjohann commented on spl_noop.so:
why push .so files ?

ananas-block replied:
I was struggling with failing tests for mock verifier.
Apparently the docker image didn’t use the local verifier program two binary.
Therefore, I changed the runTest.sh script to start a test validator without docker. But the spl noop binary has been saved in the docker image so far.
It was easier to push this than to change the docker image.

SwenSchaeferjohann replied:
okay I see, so this is probably temporary?

SwenSchaeferjohann reviewed (commented):
besides the comments lgtm! feel free to merge after addressing the comments

also generally wondering how % test coverage is here?

ananas-block commented on light-system-programs/scripts/airdropTestnet.ts:
this is how I funded the testnet deployment.
It’s not fast. The loop sleeps for 10s after every airdrop to avoid ratelimiting.

ananas-block commented:
> besides the comments lgtm! feel free to merge after addressing the comments
> 
> also generally wondering how % test coverage is here?

test coverage of what specifically?

(Most of the changes are to code which is already covered by tests and for the scripts to init a lookup table and airdrop tests are overkill in my opinion because it is a temporary solution these should be part of the cli.)
